src/main/java/hudson/tasks/junit/GroupByError.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2015, Hyunil Shin
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package hudson.tasks.junit;
+
+import hudson.tasks.test.TestObject;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * 
+ * Group case results by error message.
+ */
+public class GroupByError {
+	private final TestObject testObject;
+
+	/**
+	 * Maximum percentage of difference for testing whether two strings are similar.
+	 * If the percentage of difference of two strings is below 50%, they seem to be similar.
+	 * 0.5f (50%) is just personal opinion.
+	 */
+	private static double MAX_DIFF_PERCENTAGE_FOR_SIMILARITY = 0.5f;
```

Comment:
Thank you~

I wanted to make it configurable in the result page (GroupByError/index.jelly), not the job setting. I mean that a user can change the value and refresh to get a new result in the result page. Is that possible? Any source code for me to refer to?